[2024-05-01T15:09:55.641+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_parsing.fetch_data_pound scheduled__2024-04-30T00:00:00+00:00 [queued]>
[2024-05-01T15:09:55.656+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_parsing.fetch_data_pound scheduled__2024-04-30T00:00:00+00:00 [queued]>
[2024-05-01T15:09:55.656+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-05-01T15:09:55.675+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): fetch_data_pound> on 2024-04-30 00:00:00+00:00
[2024-05-01T15:09:55.681+0000] {standard_task_runner.py:60} INFO - Started process 966 to run task
[2024-05-01T15:09:55.687+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'currency_parsing', 'fetch_data_pound', 'scheduled__2024-04-30T00:00:00+00:00', '--job-id', '2609', '--raw', '--subdir', 'DAGS_FOLDER/stage/currency_parsing.py', '--cfg-path', '/tmp/tmpzl2q1knw']
[2024-05-01T15:09:55.690+0000] {standard_task_runner.py:88} INFO - Job 2609: Subtask fetch_data_pound
[2024-05-01T15:09:55.780+0000] {task_command.py:423} INFO - Running <TaskInstance: currency_parsing.fetch_data_pound scheduled__2024-04-30T00:00:00+00:00 [running]> on host 84be22a86186
[2024-05-01T15:09:56.090+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_parsing' AIRFLOW_CTX_TASK_ID='fetch_data_pound' AIRFLOW_CTX_EXECUTION_DATE='2024-04-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-30T00:00:00+00:00'
[2024-05-01T15:09:56.135+0000] {base.py:83} INFO - Using connection ID 'dwh' for task execution.
[2024-05-01T15:09:56.147+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/stage/currency_parsing.py", line 73, in fetch_data_pound
    dwh_hook.insert_rows(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 565, in insert_rows
    cur.execute(sql, values)
psycopg2.errors.UndefinedTable: relation "stage.currency_parsing_pound" does not exist
LINE 1: INSERT INTO stage.currency_parsing_pound (title, rate, date)...
                    ^

[2024-05-01T15:09:56.172+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=currency_parsing, task_id=fetch_data_pound, execution_date=20240430T000000, start_date=20240501T150955, end_date=20240501T150956
[2024-05-01T15:09:56.195+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 2609 for task fetch_data_pound (relation "stage.currency_parsing_pound" does not exist
LINE 1: INSERT INTO stage.currency_parsing_pound (title, rate, date)...
                    ^
; 966)
[2024-05-01T15:09:56.240+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-05-01T15:09:56.274+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-05-01T15:09:56.300+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
